contracts/TokenomicsConstants.sol

```diff
@@ -8,7 +8,7 @@ import "@prb/math/src/UD60x18.sol";
 /// @author Aleksandr Kuperman - <aleksandr.kuperman@valory.xyz>
 abstract contract TokenomicsConstants {
     // Tokenomics version number
-    string public constant VERSION = "1.0.0";
+    string public constant VERSION = "1.0.1";
```

Comment:
Are we redeploying this contract?

Reply:
Yes, it's part of the Tokenomics contract.